x-pack/platform/plugins/shared/fleet/server/services/settings.ts

```diff
@@ -72,7 +72,13 @@ export async function getSettingsOrUndefined(
 
 export async function settingsSetup(soClient: SavedObjectsClientContract) {
   try {
-    await getSettings(soClient);
+    const config = appContextService.getConfig();
+    const settings = await getSettings(soClient);
+    if (config?.prereleaseEnabledByDefault && !settings.prerelease_integrations_enabled) {
```

Comment:
This means there is no way for user to opt-out from prerelease integrations it is the expected behaviour?